### DISCUSSION TITLE
Logging out problems

### DISCUSSION THREAD

**Discussion by RiteshChepuri** [i3/i3#5878] [Q&A]
Everytime I logout of i3 and use gtk once for other uses every account in browsers gets logged out. Does anyone have any issues same as me or is it normal? Need help.

**Comment by orestisfl** [comment]
Perhaps an issue with password stores? I think this happens with chromium, see https://wiki.archlinux.org/title/chromium#Force_a_password_store

If you are using chromium, try starting it with `--password-store basic` and see if logins are preserved. Eventually, it's a good idea to set up [a keyring like the one from GNOME](https://wiki.archlinux.org/title/GNOME/Keyring).
